coderd/workspaceapps/auth.go

```diff
@@ -38,6 +39,12 @@ const (
 //
 // Upstream code should avoid any database calls ever.
 func (p *Provider) ResolveRequest(rw http.ResponseWriter, r *http.Request, appReq Request) (*Ticket, bool) {
+	// nolint:gocritic // We need to make a number of database calls. Setting a system context here
+	//                 // is simpler than calling dbauthz.AsSystemRestricted on every call.
+	// 			       // dangerousSystemCtx is only used for database calls. The actual authentication
```

Comment:
Spacing on this comment?

Reply:
Now that I look at it, it's incredibly obnoxious! 😅
I thought I was aligning things nice and neatly.